lib/matplotlib/tri/_tricontour.py

```diff
@@ -219,6 +220,9 @@ def _contour_args(self, args, kwargs):
 
 @_docstring.Substitution(func='tricontour', type='lines')
 @_docstring.interpd
+@delete_parameter("3.9", "tricontour", "origin")
+@delete_parameter("3.9", "tricontour", "extent")
+
```

Comment:
```suggestion
@_api.delete_parameter("3.9", "origin")
@_api.delete_parameter("3.9", "extent")
```
This is how you'd use the decorator. The `func` parameter is not passed in explicitly but comes from the decoration.

That said, I'm not sure whether the decorator works for parameters passed via `**kwargs`.

Reply:
Also, "3.9" should be replaced with the version it is deprecated in (right now it looks like "3.11").

Reply:
> That said, I'm not sure whether the decorator works for parameters passed via `**kwargs`.

It's fine in `**kwargs`, at least by documentation.